backend/app/api/endpoints/local_executor.py

```
@@ -16,17 +16,57 @@
 logger = logging.getLogger(__name__)
 
 router = APIRouter(tags=["local-executor"])
 
 
+@router.post(
+    "/devices/{device_id}/direct-chat/connections",
+    response_model=DirectChatConnectionResponse,
+)
+async def create_direct_chat_connection(
+    device_id: str,
+    db: Session = Depends(get_db),
```

Ruff (0.15.17)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)